### ISSUE TITLE
Feature request: shareable URLs

### ISSUE THREAD

**Issue by LandonSchropp** [exercism/exercism#4934]
Our team is doing some of the Exercism exercises as a group. It would be great if we could copy and paste the URLs on the pages so we can share them in our team Slack. Currently, there doesn't seem to be a good way to share the URLs.

**Comment by NobbZ** [comment]
Are you talking about regular mentored edition or the teams edition?

The former should have a copy-pastable URL as soon as you got an exercise approved and made it public.

I'm not sure about the latter…

**Comment by LandonSchropp** [comment]
We're all using Individual Mode. We weren't looking for mentoring features within the platform. Is that a feature of the other versions?

As an example, I was trying to share the Hamming problem in our group Slack channel, but the URL for the instructions is this: https://exercism.io/my/solutions/d98b614f27d64f6baeffc0cc73c6f460.

**Comment by LandonSchropp** [comment]
I would expect the URL for the problem instructions to be something like this: https://exercism.io/problems/javascript/hamming.

**Comment by NobbZ** [comment]
You can only publish and share mentored solutions, this is by design.

If you use the independant mode, you are not able to publish or share your solutions at all.

The [Teams-edition](https://teams.exercism.io/) might be something for you though. I have not used it, but its meant to be used by teams for internal learning experience.

**Comment by LandonSchropp** [comment]
Our goal isn't to share our solutions, it's to share the problem we're working on with each other.

**Comment by NobbZ** [comment]
So you want to share your code pre-submission? You can't do that…

The word "solution" refers to any submitted code that belongs to an exercise.

**Comment by LandonSchropp** [comment]
I'm trying to do is share the *instructions* with my team, sans any code of my own. We just want a link we can send to our slack group along like this:

> Hey everyone, this is the problem we're going to tackle this week. https://exercism.io/problems/javascript/hamming.

Before we can talk about solutions, we want a way to share the problem with the team so everyone can work on it. It's much easier to share a link to the problem then print out the instructions for tracking it down in the Exercism UI.

**Comment by LandonSchropp** [comment]
![Screen Shot 2019-07-17 at 10 42 40 AM](https://user-images.githubusercontent.com/361591/61397909-c4592400-a87f-11e9-9f84-e4ac7f132569.png)

**Comment by NobbZ** [comment]
Okay, now I understand what you are talking about.

I'll rename the issue accordingly.

**Comment by LandonSchropp** [comment]
Thanks!

**Comment by iHiD** [comment]
Hello. So just to clarify, you can't do this at the moment. It's not a request that we've had before, and while that doesn't mean we won't do it, it'll probably not happen soon as there are a bazillion things we get asked about each day (plus our own plans at https:/exercism.io/strategy)

I feel like using teams.exercism.io might work better for you here so you can more easily see what each other are doing and therefore quickly be able to start the same exercises as each other.

Either way, thanks for the idea and I'll update this if/when we get round to scheduling the feature :)
